### PR TITLE
[BACKLOG-8335] Fix styling for the error message dialog in Onyx theme

### PR DESCRIPTION
**Pull request by Aliaksandr-Kastenka**
@tmorgner 
Please review.

### CODE REVIEW

File: package-res/reportviewer/reportviewer-prompt.js [hunk 1]
```diff
@@ -344,14 +344,18 @@ define([
       showMessageBox: function( message, dialogTitle, button1Text, button1Callback, button2Text, button2Callback, blocker ) {
         var messageBox = registry.byId('messageBox');
 
-        messageBox.setTitle(dialogTitle);
-        messageBox.setMessage(message);
+        messageBox.setTitle('<div class="pentaho-reportviewer-errordialogtitle">' + dialogTitle + '</div>');
+        messageBox.setMessage('<div class="pentaho-reportviewer-errordialogmessage">' + message + '</div>');
```
Comment (tmorgner) [inline package-res/reportviewer/reportviewer-prompt.js]: Are you sure this is correct? If you'd wanted to add a style class, I would expect that you set it in the template, not here as part of the string.

Comment (Aliaksandr-Kastenka) [inline package-res/reportviewer/reportviewer-prompt.js]: I want to make it similar to:

Comment (tmorgner) [inline package-res/reportviewer/reportviewer-prompt.js]: Woha. Now there is some seriously bad coding. But OK, if you want to use raw HTML, then at least do it safely. You must properly encode the contents of "dialogTitle" and "message" or you can easily create HTML and Script injections and other nasty surprises. 

You can use the escapeHTML method of "cdf/dashboard/Utils" for that purpose.

**Comment by Aliaksandr-Kastenka** [comment]
@tmorgner 
Pull-request updated.

**Pull request merged**